server/re/CMakeLists.txt

```diff
@@ -60,6 +60,8 @@ target_include_directories(
   "$<BUILD_INTERFACE:${CMAKE_IRODS_SOURCE_DIR}/server/core/include>"
   "$<BUILD_INTERFACE:${CMAKE_IRODS_SOURCE_DIR}/server/delay_server/include>"
   PRIVATE
+  "${CMAKE_IRODS_SOURCE_DIR}/lib/administration/user/include"
+  "${CMAKE_IRODS_SOURCE_DIR}/lib/administration/zone/include"
```

Comment:
Need to think about this a little more.

Need the user administration library (which relies on a type from the zone administration) for a new microservice (i.e. `msiRemoveUserFromGroup`), hence why I added this.

I'm wonder if this should use `$<BUILD_INTERFACE:...>` and/or whether it should be in the public section.

Is there a better way to do this?

@SwooshyCueb Thoughts?

Reply:
Yeah you want `$<BUILD_INTERFACE:...>` and in the `PUBLIC` section.

Reply:
Can you explain how you arrived at that conclusion?

Reply:
Created irods/irods_docs#208 to capture CMake knowledge in the developer docs.